projects/plugins/jetpack/extensions/blocks/image-compare/view.js

```diff
@@ -320,7 +326,9 @@ domReady( function () {
 				this.slider = document.createElement( 'div' );
 				this.slider.className = 'jx-slider';
 				this.wrapper.appendChild( this.slider );
-
+				// Need to get the nearest parent document to calculate scrolltop
+				// in case the block is in an iframe.
+				this.sliderParentDocument = this.wrapper.closest( 'html' ).parentNode;
```

Comment:
I'm not sure IE (any version) [Element.closest()](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) 🤔 

Do we polyfill it?

Reply:
Okay, I retested in latest edge and it's fine. 

Only IE11 that doesn't support:

<img width="411" alt="Screen Shot 2021-04-15 at 3 24 18 pm" src="https://user-images.githubusercontent.com/6458278/114818188-afb41200-9dfe-11eb-9f6f-ca1877e46163.png">

Reply:
I checked with @apeatling and given IE 11 support is being dropped we are not worried about this.

Reply:
I think we should not worry about it.

Reply:
Found a more efficient way of getting the parent document anyway, which does by chance happen to be IE compat, so if you had time to give it another test @ramonjd, but probably no need to check IE.